test/node/bson_compliance_test.js

```diff
@@ -6,130 +6,113 @@ var BSON = require('../..'),
   ObjectID = BSON.ObjectID,
   DBRef = BSON.DBRef,
   MinKey = BSON.MinKey,
-  MaxKey = BSON.MaxKey;
+  MaxKey = BSON.MaxKey,
+  expect = require('chai').expect;
 
 var createBSON = require('../utils');
 
-/**
- * Retrieve the server information for the current
- * instance of the db client
- *
- * @ignore
- */
-exports.setUp = function(callback) {
-  callback();
-};
+describe('BSON Compliance', function() {
+  /**
+   * @ignore
+   */
+  it('Pass all corrupt BSON scenarios ./compliance/corrupt.json', function(done) {
+    // Read and parse the json file
+    var scenarios = require(__dirname + '/compliance/corrupt');
 
-/**
- * Retrieve the server information for the current
- * instance of the db client
- *
- * @ignore
- */
-exports.tearDown = function(callback) {
-  callback();
-};
+    // Create a new BSON instance
+    var bson = createBSON();
 
-/**
- * @ignore
- */
-exports['Pass all corrupt BSON scenarios ./compliance/corrupt.json'] = function(test) {
-  // Read and parse the json file
-  var scenarios = require(__dirname + '/compliance/corrupt');
+    for (var i = 0; i < scenarios.documents.length; i++) {
+      var doc = scenarios.documents[i];
+      if (doc.skip) continue;
 
-  // Create a new BSON instance
-  var bson = createBSON();
-
-  for (var i = 0; i < scenarios.documents.length; i++) {
-    var doc = scenarios.documents[i];
-    if (doc.skip) continue;
-
-    try {
-      // Create a buffer containing the payload
-      var buffer = new Buffer(doc.encoded, 'hex');
-      // Attempt to deserialize
-      bson.deserialize(buffer);
-      test.ok(false);
-    } catch (err) {
-      test.ok(true);
+      try {
```

Comment:
I think this is cleaner and more readable than the try/catch:
```
expect(()=>{
	bson.deserialize(buffer)
}).to.throw();

Reply:
@kmahar absolutely. This was more of an incremental change, but yes moving forward we should indeed use the new tools we have